spdx/file.py

```diff
@@ -43,10 +43,10 @@ class File(object):
     - type: one of FileType.SOURCE, FileType.BINARY, FileType.ARCHIVE
       and FileType.OTHER, optional zero or one.
     - chksum: SHA1, Mandatory one.
-    - conc_lics: Mandatory one. document.License or utils.NoAssert or utils.SPDXNone.
+    - conc_lics: Mandatory one. spdx.license.License or utils.NoAssert or utils.SPDXNone.
```

Comment:
Looks like we sometimes reference it as `spdx.license.License` and sometimes as `license.License`. The second option is shorter (and we already imported `license`), so I'd go with that everywhere

Reply:
You are right, I tried to follow the existing code here. As I understand it, they used `spdx.license.License` when writing an error message, so that the user -probably on top-level- could follow the path. Within the code, so in docstrings it is sufficient to use `license.License` as we already imported it. I don't have a strong opinion about the different usages and the code also isn't consistent here so I will change this for `license.License`. Nevertheless, I think it would be good to make this consistent and decide whether to use `spdx.` everywhere or not.